missouri/demographics.yml

```diff
@@ -0,0 +1,117 @@
+demographics:
+  white:
+    ppl: 0.562
+    sex_type:
+      MSM:
+        ppl: 1.0
+        sex_role:
+          init:
+            versatile: 0.380
+            insertive: 0.356
+            receptive: 0.264
+        safe_sex:
+          prob: 0.234
+        prep:
```

Comment:
In order to use race-specific PrEP, you need to change the PrEP `target_model` to `Racial`